### DISCUSSION TITLE
Call for maintainers

### DISCUSSION THREAD

**Discussion by charleskorn** [charleskorn/kaml#624] [Announcements]
As you may have noticed, I've been getting slower and slower to respond to issues and PRs over the past few months, and unfortunately the situation is unlikely to improve any time soon. 

As a result, I am looking for others to join the project as maintainers, or to take over the project as a new owner. 

If you are interested in becoming a maintainer or becoming an owner, please respond below.

**Comment by krzema12** [comment]
@charleskorn hi, [snakeyaml-engine-kmp](https://github.com/krzema12/snakeyaml-engine-kmp/)'s owner/maintainer here. I can help maintain the lib. As a maintainer, the question is what extra powers/responsibilities I'd have (I presume: accepting PRs, and you could still take care of releasing?). I already help by contributing from time to time: https://github.com/charleskorn/kaml/commits?author=krzema12 or today's https://github.com/charleskorn/kaml/pull/664. In particular, I'm happy to unclog various dependency updates, and I usually make sure that kaml updates to the most recent version of snakeyaml-engine-kmp anyway.

**Reply by charleskorn** [reply]
Thanks for your interest @krzema12!

The main thing I'd like help with is responding to PRs and issues - if you're able to help out with that, that would be great.
